test/visual/generic_rendering.js

```diff
@@ -361,5 +361,112 @@
     height: 100,
   });
 
+  function gradientStroke(canvas, callback) {
+		var line = new fabric.Line([10, 10, 200, 200], {
+			stroke: {
+				toLive(ctx) {
```

Comment:
please remove this stroke: { toLive: .... } the correct form, as you can see in other tests is:

```js
var line = new fabric.Line([10, 10, 200, 200], {
  stroke: new fabric.Gradient({ ... options })
})
```